module12/src/main/java/com/TEKcamp/module12/Controllers/UserController.java

```diff
@@ -0,0 +1,68 @@
+package com.TEKcamp.module12.Controllers;
+
+import com.TEKcamp.module12.Entities.User;
+import com.TEKcamp.module12.Repositories.UserRepository;
+import com.TEKcamp.module12.Services.UserService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController //convenience annotation instead of @Controller, includes ResponseBody
+@RequestMapping("users") //where it's listening
+
+public class UserController {
+    private final UserService userService;
+    private final UserRepository userRepository;
+
+    public UserController(UserService userService, UserRepository userRepository) {
+        this.userService = userService;
+        this.userRepository = userRepository;
+    }
+
+    public UserService getUserService() {
+        return userService;
+    }
+
+    public UserRepository getUserRepository() {
+        return userRepository;
+    }
+
+    //    Retrieving a list of all the users and returning it as a response
+    @GetMapping
+    public List<User> getUsers(){
+        return userService.getUsers();
```

Comment:
Where's the rest of the logic? You're returning a List of your entity? There's protected, sensitive information in that model that you do not want to expose. With this simple implementation you will return the email and password for every user to anyone who makes this api call